src/pages/cplusplus/Compilers.md

```diff
@@ -23,26 +23,26 @@ is likely enough to do something on your computer, it still isn't a working exec
 important step to reach an executable program.
 
 C++ contains a vast library to aid in performing difficult tasks like I/O and hardware manipulation. You can include these 
-libraries with preprocessor directives, but the preprocessor dosen't automatically add them to your code. In order for you to have 
+libraries with preprocessor directives, but the preprocessor doesn't automatically add them to your code. In order for you to have 
 a final executable program, another utility known as the linker must combine your object files with the library functions
-necessary to run the code. Think of it as havign all the necessary blocks
+necessary to run the code. Think of it as having all the necessary blocks
 to build a house. The compiler made all the blocks but the linker is the one that sticks them all together to finally create a house.
 Once this is done, you now have a functioning executable file!
 
 
 ## How to Compile a file
-Lets say you have a C++ file called `helloWorld.cpp` ...
+Let's say you have a C++ file called `helloWorld.cpp` ...
 
 ### If you are on Windows -- 
 1. Open a Developer Command Prompt - For this step, you will need to have Microsoft Visual Studio or some other IDE that 
 enables you to compile your program from the command line. You can also search online for C++ compilers.
 
-2. Navigate to the source code directy
+2. Navigate to the source code directly
 
 3. Run the Compiler on your source code (assuming you are using the Microsoft Visual Studio compiler)
 `cl /EHsc helloWorld.cpp`
 
-This will now create a object file and automatically link it for you. If you look in that same folder, you will see a 
+This will now create an object file and automatically link it to you. If you look in that same folder, you will see a 
```

Comment:
The "link it for you" was fine. So you can change it back to "...and automatically link it for you."